stl/inc/system_error

```diff
@@ -544,12 +549,16 @@ public:
     }
 
     _NODISCARD error_condition default_error_condition(int _Errval) const noexcept override {
-        // make error_condition for error code (generic if possible)
-        const int _Posv = _Winerror_map(_Errval);
-        if (_Posv == 0) {
-            return error_condition(_Errval, _STD system_category());
+        if (_Errval == 0) {
+            return error_condition(0, _STD generic_category());
         } else {

```

Comment:
There are enough lines in the `else` here that we lose symmetry; I think it would be more readable to early-return and outdent.